container/lxd/server.go

```diff
@@ -67,6 +67,16 @@ func (s *Server) UpdateContainerConfig(name string, cfg map[string]string) error
 	return errors.Trace(resp.Wait())
 }
 
-func isLXDNotFound(err error) bool {
-	return err.Error() == "not found"
+// CreateClientCertificate adds the input certificate to the server,
+// indicating that is for use in client communication.
+func (s *Server) CreateClientCertificate(cert *Certificate) error {
+	req, err := cert.AsCreateRequest()
+	if err != nil {
+		return errors.Trace(err)
+	}
+	return errors.Trace(s.CreateCertificate(req))
+}
+
+func IsLXDNotFound(err error) bool {
+	return err != nil && err.Error() == "not found"
```

Comment:
Just to ask, there's really no type check we can do? It has to be a string compare?

Reply:
The old version of the API exposed some of the HTTP primitives like response codes, which were used to transform such returns into our _NotFound_ error type. This is abstracted away in the new API; on our side of the facade all we get is a vanilla error with "not found" as the message.

On their side of the facade they have a _NotFound_ reponse type:
https://github.com/lxc/lxd/blob/e8b151eb963a0175453fb1d589c2e7cf1a2c9bc0/lxd/response.go#L497

But this is parsed into the vanilla error mentioned above.